### PR TITLE
Added italian translation

### PR DESCRIPTION
**Pull request by blastbeng**
Hi, I've added italian translations to your repository

### CODE REVIEW

File: jellyfin_alexa_skill/alexa/setup/manifest/manifest.json [hunk 1]
```diff
@@ -44,6 +45,16 @@
             "Alexa sage Jellyfin Player spiele meine Playlist 'Beste Musik'"
           ],
           "description": "Spiele Medien von deinem Jellyfin Server.\n\nQuellcode: https://github.com/infinityofspace/jellyfin_alexa_skill"
+        },
+        "it-IT": {
+          "name": "Jellyfin Player",
+          "summary": "Riproduci media dalla tuo Server Jellyfin",
+          "examplePhrases": [
+            "Alexa chiedi a Jellyfin di riprodurre la mia musica",
```
Comment (infinityofspace) [inline jellyfin_alexa_skill/alexa/setup/manifest/manifest.json:53]: We are currently replace the invocation name in the example phrases with the custom invocation name from the config file. So we expect the name in the example phrases is the same name as the default display name which is currently `Jellyfin Player`. As it now turns out, the current approach is not ideal, as this always requires `Jellyfin Player` as a string in the examples.

**This does not require any changes from your side on this problem.**
This comment is only for tracking and documentation of the problem, so we can adjust the already unreleased new changes accordingly again to this point after this PR is merged.

**Review by infinityofspace — CHANGES_REQUESTED** [review]
Thanks for your PR.

Translations are very welcome.
The changes looks good. Only the msgid and msgstr need to be adjusted accordingly, then the PR is ready for the merge and will be included in the next release.

File: jellyfin_alexa_skill/alexa/setup/manifest/manifest.json (hunk 1 above)
Comment (blastbeng) [reply jellyfin_alexa_skill/alexa/setup/manifest/manifest.json:53]: Ok thank you, i will solve the problem asap :)

**Review by infinityofspace — APPROVED** [review]
Looks good!
Thanks for your efforts to improve and expand the skill with new languages.

**Comment by blastbeng** [comment]
Hi!! I know a years is passed but could you add me to the "Contributors" ?

**Pull request merged**